app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,5 @@
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{rental-listing rental=rentalUnit goToShowRoute="goToShowRoute"}}
```

Comment:
@sbatson5 You should just wrap rental listing in a {{link-to}} like this:

```javascript
    {{#link-to "rentals.show" rentalUnit.slug}}
      {{rental-listing rental=rentalUnit}}
    {{/link-to}}
```

and remove all the "goToShowRoute" stuff